src/scanspec/regions.py

```diff
@@ -93,87 +95,6 @@ def _merge_axis_sets(axis_sets: List[Set[Axis]]) -> Iterator[Set[Axis]]:
             yield axis_set
 
 
-@dataclass(config=StrictConfig)
```

Comment:
Re-order these back to how the order was, now that we've made the changes necessary for the Spec referencing types can go at the top still